aws-customerprofiles-calculatedattributedefinition/lombok.config

```diff
@@ -0,0 +1 @@
+lombok.addLombokGeneratedAnnotation = true
```

Comment:
Is this needed externally?

Reply:
I think yes, looking at the existing providers repositories, all of them have this config file present.